CONTRIBUTING.md

```diff
@@ -1,7 +1,17 @@
 # Getting Involved
  
 Thanks for your interest in the project, we'd love to have you involved! Check out the sections below to find out more about what to do next...
- 
+## Adding a New Template
+
+If you have built an integration that connects a 3rd party service to Orbit, we welcome your contribution of an automation template in this project!
+
+For your template to be considered please make sure the following checklist is met:
+
+* [ ] The template is in a sub-folder named for the integration
+* [ ] The template has an `INSTRUCTIONS.md` file with clear steps on how to use it
+* [ ] The integration uses the following environment variable names for the Orbit credentials: `ORBIT_WORKSPACE_ID` and `ORBIT_API_KEY`
+
+Once you are ready, you can open up a new pull request with your contribution and a member of the Orbit team will review it.
```

Comment:
We could add a small sentence saying that we’re here to help if one doesn’t know how to complete one (or more) of the items here: “If you’re unsure on how to complete those items, please open an issue or reach out to devrel@orbit.love, we’d love to help you get your integration featured here!”